aps/rms_jobs/import_fields_from_disk.py

```diff
@@ -29,14 +29,12 @@
 # (including the added trends for those that should have trends) is then copied back to the geomodel grid.
 
 from pathlib import Path
-from roxar import Direction
+from rmsapi import Direction
```

Comment:
We aware this will break backwards-compatibility of older versions of RMS.

We should probably create an internal module, which tries `rmsapi` and then fall back to `roxar`.

Reply:
We can have a separate PR on handling roxar versus rmsapi